cmd/ocm/edit/machinepool/cmd.go

```diff
@@ -195,10 +196,10 @@ func run(cmd *cobra.Command, argv []string) error {
 	if args.autoscaling.Enabled {
 		asBuilder := cmv1.NewMachinePoolAutoscaling()
 
-		if args.autoscaling.MinReplicas > 0 {
+		if isMinReplicasSet {
```

Comment:
You should probably check that `isMinReplicaSet && args.autoscaling.MinReplicas >= 0`

Reply:
We don't validate inputs here. We let ocm backend do it and return the error. 
The change was just to allow sending 0 and not treat it as 'not-set'